.github/workflows/build_and_upload.yml

```diff
@@ -7,6 +7,7 @@ on:
  push:
     branches:
      - master
+     - BAH-2765
```

Comment:
@SanoferSameera can you remove the branch

Reply:
removed